cvat/schema.yml

```diff
@@ -6089,6 +6220,19 @@ components:
           maxLength: 150
       required:
       - username
+    BinaryOperation:
+      type: object
+      properties:
+        left:
+          type: string
+          nullable: true
+        operator:
+          $ref: '#/components/schemas/OperatorEnum'
+        right:
+          type: string
+          nullable: true
```

Comment:
It's not clear what the "left" and "right" strings are supposed to be. This could use some documentation.